File: Back/Apis/Api_Js/src/settings/consultas_sql/variables_sql.js
//Librerias
import { config } from "dotenv"; 
config(); 

export const base_datos_hostname = process.env.BASE_DATOS_HOSTNAME;
export const base_datos_puerto = process.env.BASE_DATOS_PUERTO;
export const base_datos_usuario = process.env.BASE_DATOS_USUARIO;
export const base_datos_contrasena = process.env.BASE_DATOS_CONTRASENA;
export const base_datos_nombre = process.env.BASE_DATOS_NOMBRE;

File: Back/Apis/Api_Py/src/settings/base_datos_sql/variables_sql.py
#Librerias Instaladas
from dotenv import load_dotenv;

#Modulos Nativos
import os;

load_dotenv();

BASE_DATOS_HOSTNAME = os.getenv("BASE_DATOS_HOSTNAME");
BASE_DATOS_PUERTO = os.getenv("BASE_DATOS_PUERTO");
BASE_DATOS_USUARIO = os.getenv("BASE_DATOS_USUARIO");
BASE_DATOS_CONTRASENA = os.getenv("BASE_DATOS_CONTRASENA");
BASE_DATOS_NOMBRE = os.getenv("BASE_DATOS_NOMBRE");
